### ISSUE TITLE
Fam, you had stolen code from CatServer even after they had made it closed-source. You were removed for being a liability.

### ISSUE THREAD

**Issue by Mgazul** [MatrixDevTeam/LavaBukkit#81]
Fam, you had stolen code from CatServer even after they had made it closed-source. You were removed for being a liability.

_Originally posted by @GMatrixGames in https://github.com/MatrixDevTeam/LavaBukkit/issues/80#issuecomment-468960604_

steal? I developed the branch according to the GNU General Public License v3.0 protocol, and later they closed the source. Your nms repair is also stealing the source of the catserver. In China catserver has a paid version (by selling catserver to make money)

**Comment by GMatrixGames** [comment]
I have permission for CatServer's dev to use it

**Comment by GMatrixGames** [comment]
You don't, so you actually can't be using their code in Mohist...

**Comment by Mgazul** [comment]
Mohist I am re-made, does not contain catserver stuff

**Comment by GMatrixGames** [comment]
https://cdn.discordapp.com/attachments/551449368763432962/551458884259282965/unknown.png

**Comment by Mgazul** [comment]
Because of the competitive relationship, the relationship between Mohist and Catserver has been very rigid, the induction of money, Mohist is a public welfare project.

**Comment by Luohuayu** [comment]
别丢人丢到国外了

**Comment by TheZihanGu** [comment]
Plagiarism is a shameful act... and a violation of the open source protocol used.

**Comment by Luohuayu** [comment]
重制?自己心里有b数么?

**Comment by Luohuayu** [comment]
Mohist is shit

**Comment by TheZihanGu** [comment]
Change the package name? I think LavaBukkit can be made by using Visual Studio Code and renaming

**Comment by KomiMoe** [comment]
https://s2.ax1x.com/2019/03/03/kL2sdP.jpg
translate:
A: Are the Mohist is only development by you?
Mgazul: YES!

**Comment by ItsDia** [comment]
The world is peaceful and lovely

**Comment by Luohuayu** [comment]
> 
> 
> Because of the competitive relationship, the relationship between Mohist and Catserver has been very rigid, the induction of money, Mohist is a public welfare project.

https://github.com/PFCraft/Mohist/commit/a95909e5bd02fef043927120d4ad4564fc6754c6#r32561890
Why are you secretly removing code comment?

**Comment by KomiMoe** [comment]
https://github.com/PFCraft/Mohist/commit/a95909e5bd02fef043927120d4ad4564fc6754c6#r32561890

Mgazul committed 5 hours ago

Search CatServer
哎呀妈呀，CatServer真香

**Comment by Luohuayu** [comment]
https://github.com/PFCraft/Mohist/commit/a95909e5bd02fef043927120d4ad4564fc6754c6#r32561937

**Comment by liach** [comment]
Search atom here. https://github.com/MatrixDevTeam/LavaBukkit/search?q=atom&unscoped_q=atom

**Comment by KomiMoe** [comment]
Atom is MIT

**Comment by josephworks** [comment]
Yes, plus we only use atom code in one file that will be in the released jar (https://github.com/MatrixDevTeam/LavaBukkit/blob/3792de649239c20528c4c352f2e3e0c84d4245ae/src/main/java/net/minecraft/block/BlockFire.java). The rest of the code is just for Gradle building.

**Comment by josephworks** [comment]
And plus Mgazul, how is this a LavaBukkit related issue? How will this issue even help the LavaBukkit project. Couldn't you talk to GMatrixGames in person instead of creating an issue in an issue tracker that is mentioned for code related issues?
